config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```diff
@@ -57,7 +57,7 @@ presubmits:
           - name: GINKGO_FOCUS
             value: "Workload cluster creation"
           - name: GINKGO_SKIP
-            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the external cloud provider"
+            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the in-tree cloud provider"
```

Comment:
since we're not going to backport the change (I think?) we shouldn't change the jobs targetting release branches, only the main branch job + periodic jobs targeting main

Reply:
Makes sense.

Does a future 1.* release count as a "backport" in this way? (i.e., we will make it a point not to cherry-pick any template changes at any point in the future)

Reply:
good question. for v1beta1 it does since the job is using a regex for the branch `^release-1.*` which would include any future `1.*` release branches. For the v1beta1 job we can maybe make the change additive and skip both "in tree" and "external" so that it keeps working for now and then skips the in-tree template once we cut a new release that includes the change (it won't skip external anymore since that won't be an explicit external test anymore), WDYT ?

Reply:
sgtm, done